metomi/isodatetime/datetimeoper.py

```diff
@@ -21,10 +21,8 @@
 import os
 import time
 
-from .data import (
-    Calendar,
-    get_timepoint_for_now as now2point,
-    get_timepoint_from_seconds_since_unix_epoch as seconds2point)
```

Comment:
I think `seconds2point` not being used here any more, can be removed later? I think it should be used in tests, but I guess that can be removed too?

For later I guess.